its/src/test/java/com/sonar/it/jenkins/orchestrator/JenkinsOrchestrator.java

```diff
@@ -218,8 +223,25 @@ public JenkinsOrchestrator newFreestyleJobWithSonar(String jobName, File project
     findElement(buttonByText("Save")).click();
     return this;
   }
+  
+  public JenkinsOrchestrator newFreestyleJobWithMaven(String jobName, File projectPath, String branch) {
```

Comment:
Could be renamed newFreestyleJobWithStandardMavenStep because I first though it was with Maven publisher